app/scripts/containers/config.js

```diff
@@ -0,0 +1,102 @@
+import React, {Component} from 'react'
+import {connect} from 'react-redux'
+import i18n from '../utils/i18n.js'
+
+import ConfigText from '../components/config-text'
+import ConfigButtons from '../components/config-buttons'
+import {
+  Row, Col
+}
+from 'react-bootstrap'
+
+import {
+  config as configActions,
+  pages,
+  errors
+} from '../actions'
+
+const {config} = configActions
+
+export
+default class ConfigContainer extends Component {
+  static displayName = 'Config';
+  static propTypes = {
+    // Dispatch
+    loadConfigPage: React.PropTypes.func.isRequired,
+    resetErrorMessage: React.PropTypes.func.isRequired,
+    saveConfigClick: React.PropTypes.func.isRequired,
+    saveConfigDraft: React.PropTypes.func.isRequired,
+    savedConfig: React.PropTypes.func.isRequired,
+    resetConfigDraft: React.PropTypes.func.isRequired,
+    setHeight: React.PropTypes.func.isRequired,
+    // State
+    config: React.PropTypes.string.isRequired,
+    draft: React.PropTypes.string.isRequired,
+    savingConfig: React.PropTypes.bool.isRequired,
+    saved: React.PropTypes.bool.isRequired,
+    height: React.PropTypes.bool.isRequired
+  };
+
+  componentWillMount () {
+    this.props.loadConfigPage()
+  }
+
+  componentWillUnmount () {
+    this.props.resetErrorMessage()
+    this.props.savedConfig(false)
```

Comment:
These could be dispatched via a saga on a `leaveConfigPage` action that would make it more consistent with the rest